com.elsevier.MendeleyDesktop.appdata.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.elsevier.MendeleyDesktop.desktop</id>
+  <name>Mendeley</name>
+  <developer_name>Mendeley Desktop</developer_name>
+  <summary>Mendeley is a free reference manager and an academic social network</summary>
+  <description>
+    <p>
+      With the Mendeley Reference Manager, you can easily organize and search your personal library, annotate documents and cite as you write.
+      Import papers and other documents from your desktop, your existing libraries or websites with ease. Mendeley automatically captures
+      information like authors, title and publisher, which makes organization and browsing easy.
+    </p>
+    <p>
+      Over 6 million researchers trust Mendeley to share ideas, see what’s new in their field and discuss important developments. Discover
+      new connections by searching for people on Mendeley and finding researchers with similar interests via Public Groups. You can also
+      follow new people from suggestions in your Mendeley feed.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>http://static.blog.mendeley.com/wp-content/uploads/2012/06/1.6-dev2-mac.png</image>
+    </screenshot>
+  </screenshots>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://www.mendeley.com</url>
+  <!-- These are the icons for the android  apps, from https://play.google.com/store/apps/details?id=com.mendeley -->
+  <icon type="remote" height="300" width="300">https://lh3.googleusercontent.com/KdvN1UJWNw6sBlH80T3RO01SM0OmDRlli_vq9ShVEL6SAoekKVmAPAoQGgfF8CFrCtc=w300</icon>
```

Comment:
you need 128 and 64 icons as well - (good news you can just change the w parameter on the end of the URL!)